debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlSnapshotChangeEventSource.java

```diff
@@ -258,7 +253,9 @@ protected void releaseDataSnapshotLocks(RelationalSnapshotContext<MySqlPartition
                 }
```

Comment:
Shouldn't similar code above ( https://github.com/debezium/debezium/pull/5073/files#diff-63840528f25c3335e78d8b91c2ea67602750af012c1620d9981f11cb476b480aR144-R154 )  not also use the new filter code? Maybe it must even be merged into the new method to handle only "readable databases". Whatever this is....